library/src/main/java/com/mapzen/android/DI.java

```diff
@@ -1,4 +1,4 @@
-package com.mapzen.android.dagger;
+package com.mapzen.android;
```

Comment:
Nevermind now I see why. Its because `MapDataManager` and `MapStateManager` are now package private like we discussed ;)

Still based on the number of classes accumulating in the `com.mapzen.android` package I'm thinking we need a better way to organize things.

Perhaps we can add a package for each major feature/component. Something like this:

`com.mapzen.android.graphics` - Tangram related classes
`com.mapzen.android.search` - Pelias related classes
`com.mapzen.android.routing` - Valhalla related classes
`com.mapzen.android.location` - LOST related classes

Then all the common classes like `MapzenMap`, API key related stuff, and dependency injection could remain in `com.mapzen.android` or we could create a `com.mapzen.android.core` package.

Thoughts?